packages/waas/src/auth.ts

```diff
@@ -334,7 +336,20 @@ export class SequenceWaaS {
     }
   }
 
+  private async updateTimeDrift() {
+    if (getTimeDrift() === undefined) {
+      const res = await fetch(`${this.config.rpcServer}/status`)
+      const date = res.headers.get('Date')
+      if (!date) {
+        throw new Error('missing Date header value')
```

Comment:
change error to, `failed to get Date header value from /status` ... and probably better is to have a well-defined error in the waas ridl for any error type.. but the above message will do for now